### ISSUE TITLE
No message if JSON is malformed

### ISSUE THREAD

**Issue by w0rmr1d3r** [apihero-run/jsonhero-web#62]
I used https://jsonhero.io to take a look at a dirty JSON, and it seems that the JSON contained a malformed field.
Until I cleaned that field I wasn't able to put it in the input to get it read.
However, no matter how many times I tried, it didn't said anything related to an error in the page, as if it wasn't working.

It happened to me in both Safari and Chrome.

A similar behaviour happens in https://github.com/jsonhero-io/jsonhero-web/issues/42 that no answer is given.

Reviewed the issues in the repo and found nothing (besides the other issue referenced) related to this.

Thank you :)

**Comment by w0rmr1d3r** [comment]
Hello @ericallam 👋 
Sorry, but I cannot share the JSON, since it includes private data.
However, the malformation (detected by de IDE) was that instead of double quoting keys and values, it was single quoted. Such as:
```json
{
  'key': 'value'
}
```
Instead of:
```json
{
  "key": "value"
}
```

At the same time, it included some Python values without quotes.
In this case, the `key` already had double quotes since it received the fix from above:
```json
{
  "key": True
}
```
Since this is more related to Python than to jsonhero, we can ignore it for the moment.

Hope this information helps :)

**Comment by w0rmr1d3r** [comment]
Hi @ericallam ,
I see the issue has been closed, but the linked PR is closed, not merged.
Has this been solved then? Which PR fixed it?

Thank you

**Comment by ericallam** [comment]
Good catch, this shouldn't have been closed because this PR that is addressing the issue is still open:

https://github.com/apihero-run/jsonhero-web/pull/102
